### ISSUE TITLE
RequireEmptyLineBeforeBlockTagGroup gives error when theres a single trailing whitespace after asterisk

### ISSUE THREAD

**Issue by erlendps** [checkstyle/checkstyle#10860]
```bash
/var/tmp $ javac Test.java

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="RequireEmptyLineBeforeBlockTagGroup"/>
  </module>
</module>

/var/tmp $ cat Test.java
public class Test {
  /**
   * Place holder method
   * 
   * @param string given string
   */
  public void placeholder(String string) {
    // some logic
  }
}

/var/tmp $ java -jar checkstyle-9.0.1-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /var/tmp/Test.java:6: Javadoc tag '@param' should be preceded with an empty line. [RequireEmptyLineBeforeBlockTagGroup]
Audit done.
Checkstyle ends with 1 errors.
```

---

Expected that this should not give an error.

If you have one whitespace after the asterisk (*) on the preceding line of a BlockTagGroup, checkstyle gives an error stating the line is not empty. It will resolve itself if I remove the one trailing whitespace, but in the editor I use (VSC), when writing javadoc, it will automatically give a single whitespace after the asterisk when entering a new line. It is a bit annoying having to delete this one trailing whitespace everytime.

Hope you can take a look at this!

---
